libcnb-cargo/tests/integration_test.rs

```diff
@@ -219,6 +219,47 @@ fn package_command_error_when_run_in_project_with_no_buildpacks() {
     );
 }
 
+#[test]
+#[ignore = "integration test"]
+fn package_command_respects_ignore_files() {
+    let fixture_dir = copy_fixture_to_temp_dir("multiple_buildpacks").unwrap();
+
+    // when a git folder is not present, .ignore should be used
+    let ignore_file = fixture_dir.path().join(".ignore");
+    fs::write(&ignore_file, "meta-buildpacks\nbuildpacks\n").unwrap();
+
+    let output = Command::new(CARGO_LIBCNB_BINARY_UNDER_TEST)
+        .args(["libcnb", "package", "--release"])
+        .current_dir(fixture_dir.path())
+        .output()
+        .unwrap();
+
+    assert_ne!(output.status.code(), Some(0));
+    assert_eq!(
+        String::from_utf8_lossy(&output.stderr),
+        "🚚 Preparing package directory...\n🖥\u{fe0f} Gathering Cargo configuration (for x86_64-unknown-linux-musl)\n🏗\u{fe0f} Building buildpack dependency graph...\n🔀 Determining build order...\n❌ No buildpacks found!\n"
+    );
+
+    fs::remove_file(ignore_file).unwrap();
+
+    // when a git folder is not present, .gitignore can be used
```

Comment:
Should this read `.gitignore` _cannot_ be used? Otherwise I can't tell what this comment means.

Reply:
sorry, should have read "when a git folder is present...".  actually, i'll expand on both comments in this test with more detail around why.